### ISSUE TITLE
Cannot run with bhttp

### ISSUE THREAD

**Issue by kjlaw89** [igorklopov/enclose#28]
I can compile my application fine, but when I try to run it this happens:

``` javascript
fs.js:500
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/thebox/home/kevin/Documents/Projects/NIM/node_modules/bhttp/node_modules/errors/lib/static/error.css'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.readFileSync (fs.js:352:15)
    at Object.Math.Enclose.+s+t+u+b+:+/+/+f+s.fs.readFileSync (<anonymous>)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+k+e+v+i+n+/+D+o+c+u+m+e+n+t+s+/+P+r+o+j+e+c+t+s+/+N+I+M+/+n+o+d+e+_+m+o+d+u+l+e+s+/+b+h+t+t+p+/+n+o+d+e+_+m+o+d+u+l+e+s+/+e+r+r+o+r+s+/+l+i+b+/+e+r+r+o+r+s+.+j+s.dev (<anonymous>)
    at u (<anonymous>)
    at O (<anonymous>)
    at c (<anonymous>)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+k+e+v+i+n+/+D+o+c+u+m+e+n+t+s+/+P+r+o+j+e+c+t+s+/+N+I+M+/+n+o+d+e+_+m+o+d+u+l+e+s+/+b+h+t+t+p+/+n+o+d+e+_+m+o+d+u+l+e+s+/+e+r+r+o+r+s+/+i+n+d+e+x+.+j+s.dev (<anonymous>)
    at u (<anonymous>)
    at O (<anonymous>)
    at c (<anonymous>)
    at Object.Math.Enclose.+/+t+h+e+b+o+x+/+h+o+m+e+/+k+e+v+i+n+/+D+o+c+u+m+e+n+t+s+/+P+r+o+j+e+c+t+s+/+N+I+M+/+n+o+d+e+_+m+o+d+u+l+e+s+/+b+h+t+t+p+/+l+i+b+/+b+h+t+t+p+.+j+s.dev (<anonymous>)
    at u (<anonymous>)
    at O (<anonymous>)
    at c (<anonymous>)
```

I'm not entirely sure what's happening here. The "thebox" in the path does not exist, but after that the rest of the path is correct. The application runs just fine if you call it normally.

https://github.com/joepie91/node-bhttp

**Comment by igorklopov** [comment]
Fixed in 0.2.27. Please try again.

**Comment by kjlaw89** [comment]
That did indeed fix the issue, thanks!
